src/CartingService/Controllers/v2/CartController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using CartingService.Application.Cart.Models;
+using CartingService.Application.Cart.Queries.GetCart;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CartingService.Controllers.v2;
+
+[ApiController]
+[Route("api/v{version:apiVersion}/cart")]
+[ApiVersion("2.0")]
+public class CartController : ApiControllerBase
```

Comment:
Cart controller version two should also support all other actions
![CleanShot 2022-10-26 at 22 42 52@2x](https://user-images.githubusercontent.com/15052879/198121542-cdd7bd6f-389d-437f-a3aa-a43adec7c0b6.png)

Reply:
But, Is it really need to move all methods from v1 to v2? Because we just are making changes to the "Get cart info" method? So, only mentioned action will be different from v1. Everything else will be identical.

Reply:
So, you are expecting clients of your API to use different versions for the different endpoints? 

One of the solutions can be found here:
https://baldur.gitbook.io/net-c/versioning-api-mongodb-and-ci-cd/rest-api-versioning-with-asp.net-core#version-specific-controllers-and-actions

Reply:
It depends on the case: if we are going to depreciate the v1 API. So, in this case, we will move all endpoints from v1 to v2 and add changes only to interested endpoints. Ann will notify users that v1 will be deprecated soon and they should move to v2.
But if we are going to change some endpoints - change input parameters or response - we could create a new version only for these endpoints. So, users will be able to use v1 and v2 (with some changes).

Reply:
Added endpoints to v2 and marked v1 as deprecated.